apps/api/test/services/test-wallet.service.ts

```diff
@@ -62,15 +62,16 @@ export class TestWalletService {
 
     const configs = await Promise.all(
       specPaths.map(async path => {
-        const wallet = new Wallet();
+        const hdWallet = await DirectSecp256k1HdWallet.generate(24, { prefix: WALLET_ADDRESS_PREFIX });
+        const wallet = new Wallet(hdWallet.mnemonic);
```

Comment:
generate mnemonic separately, so then we can get rid of `getMnemonic` method on `Wallet` class